doc/melange.rst

```diff
@@ -0,0 +1,295 @@
+.. _melange_main:
+
+***********************************
+JavaScript Compilation With Melange
+***********************************
+
+Introduction
+------------
+
+`Melange <https://github.com/melange-re/melange>`_ is a compiler from OCaml to
+JavaScript. Unlike js_of_ocaml, Melange works by translating OCaml compiler
+internal lambda representation to JS files. This allows to produce a single
+JavaScript file from each OCaml module. Melange can be installed with
+`opam, Esy, or Nix <https://github.com/melange-re/melange#installation>`_
+package managers.
```

Comment:
```suggestion
JavaScript. Unlike Js_of_ocaml, Melange works by translating OCaml compiler
internal Lambda representation to JS files. This allows to produce a single
JavaScript file from each OCaml module. Melange can be installed with
`opam, Esy, or Nix <https://github.com/melange-re/melange#installation>`_
package managers.
```

Reply:
I think js_of_ocaml should be monospace and the lambda should be lower case since it is the textual version of the lower case lambda.

Reply:
@Alizter Js_of_ocaml is written like this (capital J, regular font) everywhere, I think the change you suggest could be done, but on isolation in a different PR.